### ISSUE TITLE
Date range picker witout dialog

### ISSUE THREAD

**Issue by mr-africa** [praveenpuglia/vuetify-daterange-picker#57]
Could you move your awesome daterangepicker component to separately component please.
I want to use it without v-dialog

**Comment by jogoetha** [comment]
It was like this until version 2.6.2, so I'm using that version in order to use it without dialog. The dialog should be optional in my opinion.

**Comment by praveenpuglia** [comment]
I agree it should be separate but I also considered how most people wanted to use it. While making v3, I first addressed the most common use case. 

I am now working on refactoring things into separate components. Will keep you updated.
